fdbserver/CMakeLists.txt

```diff
@@ -187,6 +187,18 @@ add_library(fdb_sqlite STATIC
   sqlite/sqliteLimit.h
   sqlite/sqlite3.amalgamation.c)
 
+if (WITH_ROCKSDB_EXPERIMENTAL)
```

Comment:
Should we split this change off into a separate PR? No Rocks code is used in 6.2, so this could hypothetically waste time building RocksDB for it to never be used. I guess the `WITH_ROCKSDB_EXPERIMENTAL` option will always be disabled on 6.2 builds, so the overhead is small.

Reply:
We would like to keep this code in order to allow the same build image to have the capability to build both 6-2 and 6-3. Much easier for users, developers, and administrators to use/maintain a single image that have to switch to a different image for each version when not needed.

Reply:
In theory, we could pull this part out into 6.3, but I'm OK with leaving it in.  With it, if you explicitly ask for experimental rocksdb in 6.2, the build will fail when it tries to include the CompileRocksDB stuff.  That's arguably a feature.